### ISSUE TITLE
Routes

### ISSUE THREAD

**Issue by PLUCS2** [aklee2106/Yabber#3]
Backend: 
- [ ] New post in specific group needs a `group_id`
- [ ] Do you have a way of grouping type of post? 

Frontend: 
- [ ] What is the redirect in the login?
- [ ] Add a route for editing a user profile 
- [ ] The post form is just on the page, it doesn't need its own route

**Comment by aklee2106** [comment]
> Do you have a way of grouping type of post?

Yes- on the main feed, I have buttons to filter posts by 'All' and 'Following'. 
Following = only the posts that belong to groups that the user is part of. 

> What is the redirect in the login?
Yammer redirects to Microsoft page to log in. I just got rid of it.

**Comment by PLUCS2** [comment]
Both of your `/api/group/posts` should go `api/group/:group_id/post`
